pkg/sql/information_schema.go

```diff
@@ -410,9 +410,13 @@ CREATE TABLE information_schema.table_constraints (
 	CONSTRAINT_CATALOG STRING NOT NULL DEFAULT '',
 	CONSTRAINT_SCHEMA STRING NOT NULL DEFAULT '',
 	CONSTRAINT_NAME STRING NOT NULL DEFAULT '',
+	TABLE_CATALOG STRING NOT NULL DEFAULT '',
 	TABLE_SCHEMA STRING NOT NULL DEFAULT '',
 	TABLE_NAME STRING NOT NULL DEFAULT '',
-	CONSTRAINT_TYPE STRING NOT NULL DEFAULT ''
+	CONSTRAINT_TYPE STRING NOT NULL DEFAULT '',
+	IS_DEFERRABLE STRING NOT NULL DEFAULT '',
+	INITIALLY_DEFERRED STRING NOT NULL DEFAULT ''
+
```

Comment:
Would you also mind removing this extra line? Sorry for not noticing the first time around.

Reply:
Oh, sure. Fixed.